config/manifest/eksa-components.yaml

```diff
@@ -5333,6 +5333,18 @@ metadata:
   creationTimestamp: null
   name: eksa-manager-role
 rules:
+- apiGroups:
```

Comment:
should we have the manifest changes in a separate pr?

Reply:
happy to: https://github.com/aws/eks-anywhere/pull/4023